ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -394,19 +397,29 @@ class FamilyReads extends React.PureComponent {
       showReads={this.showReads}
     />
 
-    const igvSampleIndividuals = this.state.openFamily && (igvSamplesByFamilySampleIndividual || {})[this.state.openFamily]
-    const reads = igvSampleIndividuals ?
+    const igvSampleIndividuals = (this.state.openFamily && (igvSamplesByFamilySampleIndividual || {})[this.state.openFamily]) || {}
+    const dnaTrackOptions = DNA_TRACK_TYPE_OPTIONS.filter(({ value }) => igvSampleIndividuals[value])
+    const rnaTrackOptions = RNA_TRACK_TYPE_OPTIONS.filter(({ value }) => igvSampleIndividuals[value])
+    const reads = Object.keys(igvSampleIndividuals).length ?
       <Segment.Group horizontal>
-        {Object.keys(igvSampleIndividuals).length > 1 &&
-          <Segment>
+        <Segment>
+          { dnaTrackOptions &&
```

Comment:
1) empty lists are "truthy" in javascript so I don't think this is the behavior you want, it will render the checkbox group without any options instead of rendering nothing
2) While the entire `reads` component is shown if there are any sample types, before this change the Segment showing the control panel used to be shown only if there was > 1 sample type. But now, the segment will always be shown, which is not the behavior we want

Reply:
Thanks for #1. I will correct it. We always want the control panel for the RNA tracks. Does the RNA always have two sample types (both coverage/wig and junction/spliceJunctions)? If yes, I may keep the condition `Object.keys(igvSampleIndividuals).length > 1` for displaying the control panel.

Reply:
The behavior for whether or not to show the control panel is either if there are more than one sample type OR if there are any RNA sample types. So for example, for the many samples in seqr that have just crams and nothing else there should be no control panel at all. The behavior does need to change here, because even if there is only one sample type, if its an RNA type we need to show the reference track options